src/pyeconet/equipment/water_heater.py

```diff
@@ -137,7 +137,11 @@ def tank_hot_water_availability(self) -> Union[int, None]:
     def shutoff_valve_open(self) -> Union[bool, None]:
         """Return if the shutoff valve is open or not"""
         if self.has_shutoff_valve:
-            return self._equipment_info.get("@VALVE")["value"] == 0
+            status = self._equipment_info.get("@VALVESTATUS")["title"]
```

Comment:
Same here thinking maybe doing an `or` would be safer in case different units are reporting in different ways.